lib/benchee/formatters/html.ex

```diff
@@ -263,6 +266,22 @@ defmodule Benchee.Formatters.HTML do
     end
   end
 
+  defp inline_stylesheets, do: inline_assets("stylesheets", "style")
+  defp inline_javascripts, do: inline_assets("javascripts", "script")
+
+  defp inline_assets(type, tag_name) do
+    stylesheets_dir = Application.app_dir(:benchee_html, "priv/assets/#{type}/")
+
+    stylesheets_dir
```

Comment:
it's not only stlesheets any longer but assets :)

Reply:
that's true :)